notebook/log.py

```diff
@@ -46,5 +50,11 @@ def log_request(handler):
     if status >= 500 and status != 502:
         # log all headers if it caused an error
         log_method(json.dumps(dict(request.headers), indent=2))
-    log_method(msg.format(**ns))
+    if _enable_json_logs:
+        # FIXME: this still logs the msg as a serialized json string,
```

Comment:
I wonder if we could use a partial to pass `log` in here:

https://github.com/jupyter/notebook/blob/b617d910521d7ffb5d9099b2307c6026dca4f3da/notebook/notebookapp.py#L269

Because that'd be the `NotebookApp` logger which is using our json log formatter.